recipes/pylev/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pylev" %}
+{% set version = "1.3.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "063910098161199b81e453025653ec53556c1be7165a9b7c50be2f4d57eae1c3" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - setuptools
```

Comment:
Since switching to `pip`, you need to make sure `pip` is listed under the `host:requirements`.  Remove `setuptools` and replace it with `pip`.